etc/DependencyInstaller.sh

```diff
@@ -34,6 +34,14 @@ _installCommon() {
     else
         pip3 install --no-cache-dir --user -U $pkgs
     fi
+
+    # Autotuner reqs (use venv)
+    local autotuner_pkgs=" $(cat ./etc/requirements.txt)"
```

Comment:
don't put in dependency installer.  this forces the user to download autotuner deps. 

Setup a separate script just for autotuner reqs.  -> called optionally by the user. Put inside `autotuner` folder.